packages/nuxt/src/runtime/plugins/sentry.server.ts

```diff
@@ -14,9 +14,17 @@ export default defineNitroPlugin(nitroApp => {
       }
     }
 
+    const currentScope = Sentry.getCurrentScope();
+
+    const { method, path } = {
+      method: errorContext.event && errorContext.event._method ? errorContext.event._method : '',
+      path: errorContext.event && errorContext.event._path ? errorContext.event._path : 'unknown-path',
+    };
+    currentScope.setTransactionName(`${method} ${path}`);
```

Comment:
m: I recommend not setting the transaction name at all if the path is unknown, because we will give other sensible default mechanisms to activate and show in the product, ie. root span name, or stack trace.